purchases-js-hybrid-mappings/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "purchases-js-hybrid-mappings",
+  "version": "1.0.6",
+  "description": "RevenueCat's JavaScript hybrid mappings for purchases-js",
+  "main": "dist/index.js",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "author": "RevenueCat, Inc.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/RevenueCat/purchases-hybrid-common.git"
+  },
+  "bugs": {
+    "url": "https://github.com/RevenueCat/purchases-hybrid-common/issues"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "jest",
+    "api-test": "npm run build && api-extractor run --local --verbose",
+    "api-test:ci": "npm run build && api-extractor run --verbose",
+    "prepack": "npm run build && api-extractor run --local",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix"
+  },
+  "dependencies": {
+    "@revenuecat/purchases-js": "^1.1.0"
```

Comment:
purchases-js 1.1.0 hasn't been released yet. Tests will fail until then, but we need the changes there to have the full mappings needed here (tested locally while pointing to a local version of the package)